### ISSUE TITLE
Simple Sample for usage with Monaco Editor over Websockets

### ISSUE THREAD

**Issue by jogibear9988** [OmniSharp/csharp-language-server-protocol#505]
Is there a Simple sample how to use Omnisharp with Monaco Code Editor in a Web Application.

For mee it's not clear wich Nugets I had to use, wich Handlers I need to implement.

I'd like to get simple C# Code Completition in a Web Application.

**Comment by David-Moreira** [comment]
Also curious about this!

**Comment by janivo** [comment]
Hey @jogibear9988, did you find a way to implement it?

**Comment by jogibear9988** [comment]
No, not yet. But I also don't had time to work on this. I think if noone helps here I will come back to this later this year

**Comment by jogibear9988** [comment]
And if I will get it working, I will also create a sample

**Comment by tritrancs** [comment]
Any of use get any progress with this @jogibear9988 @janivo @David-Moreira?
I have the same need.

**Comment by jogibear9988** [comment]
nope, not really.
we now started to use roslyn directly, but we do not have all of the features. but at least this works for us atm.
I will look if we could share the solution. Maybe we could move this to an opensource github library, so it could grow if others contribute something.

**Comment by jogibear9988** [comment]
we have created a repo with the code we use atm.
this still need's a lot of work .
for example, hover informations is only less usefull...
we will also create a nuget, once net6 is released.

see here:
https://github.com/dotnetprojects/MonacoRoslynCompletionProvider

**Comment by jogibear9988** [comment]
also we only handle "." at the moment.
we still need to handle brackets, show method definitions, ...
so if anyone would help.

**Comment by MaryanneNjeri** [comment]
Same here any samples with omnisharp with moncao ?
